From 63ec8e94fc849d0193ca90f1d55c5d5b43ac47e2 Mon Sep 17 00:00:00 2001
From: Aaron Ruby <aruby@qnx.com>
Date: Thu, 6 Mar 2025 14:18:11 -0500
Subject: [PATCH] gfxstream: Downgrade log severity when enabling params in
 LinuxVirtGpu

Reviewed-By: Gurchetan Singh <gurchetansingh@google.com>
Part-of: <https://gitlab.freedesktop.org/mesa/mesa/-/merge_requests/33944>
(cherry picked from commit 9447de5dc4921faf9de891b6698938fb58c1753a)
---
 .pick_status.json                                         | 2 +-
 src/gfxstream/guest/platform/linux/LinuxVirtGpuDevice.cpp | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/gfxstream/guest/platform/linux/LinuxVirtGpuDevice.cpp b/src/gfxstream/guest/platform/linux/LinuxVirtGpuDevice.cpp
index 533a14f0338..9eb6b64ec6a 100644
--- a/src/gfxstream/guest/platform/linux/LinuxVirtGpuDevice.cpp
+++ b/src/gfxstream/guest/platform/linux/LinuxVirtGpuDevice.cpp
@@ -72,7 +72,7 @@ LinuxVirtGpuDevice::LinuxVirtGpuDevice(enum VirtGpuCapset capset, int32_t descri
 
         ret = drmIoctl(mDeviceHandle, DRM_IOCTL_VIRTGPU_GETPARAM, &get_param);
         if (ret) {
-            mesa_logi("virtgpu backend not enabling %s", params[i].name);
+            mesa_logd("virtgpu backend not enabling %s", params[i].name);
             continue;
         }
 
-- 
2.45.2

